.clang-files

```diff
@@ -29,8 +31,6 @@
 ./src/common/VerificationUtils.cc
 ./src/common/VerificationUtils.h
 
-./src/interpolation/InterpolationUtils.h
-
 ./src/itehandler/IteHandler.cc
```

Comment:
Shouldn't we use the `interpolation` directory e.g. for `InterpolationContext`?

Reply:
Probably. But also not necessarily.